ansible.properties

```diff
@@ -1,15 +1,27 @@
 category=External Analyzers
 description=Analyze Ansible playbooks.
 homepageUrl=https://github.com/sbaudoin/sonar-ansible
-archivedVersions=1.0.0,2.0.0
-publicVersions=2.1.0
+archivedVersions=1.0.0,2.0.0,2.1.0,2.2.0
+publicVersions=2.3.0
  
 defaults.mavenGroupId=com.github.sbaudoin
 defaults.mavenArtifactId=ansible
 
+2.3.0.description=Support for SonarQube 8
+2.3.0.date=2019-12-23
+2.3.0.sqVersions=[7.9,LATEST]
```

Comment:
This description says "Support for SQ 8". Should compatibility start there?

Reply:
7 and 8 are both supported with this version. I meant here that this is the first version that supports SQ 8 but not exclusively.